pkgs/servers/monitoring/alertmanager-bot/default.nix

```diff
@@ -17,6 +17,8 @@ buildGoModule rec {
     sed "s;/templates/default.tmpl;$out/share&;" -i cmd/alertmanager-bot/main.go
   '';
 
+  buildFlagsArray = "-ldflags=-X main.Version=v${version} -X main.Revision=${src.rev}";
+
```

Comment:
```suggestion
  preBuild = ''
    buildFlagsArray +=(
      "-ldflags=-s -w -X main.Version=v${version} -X main.Revision=${src.rev}"
    )
  '';

```

Reply:
@SuperSandro2000 In all of the packages using `buildGoModule` in `nixpkgs` I've looked through `buildFlagsArray` is an attribute of the `buildGoModule`. Could you please explain why you are suggesting to put it into `preBuild`? Is this some new convention?

Reply:
See https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-flags-array.md and https://github.com/NixOS/nixpkgs/blob/96f048e3d07c92f86ba85c17438ca7ba963f090f/pkgs/applications/misc/writefreely/default.nix#L20

Reply:
Thanks!